dev/CHERRY_PICK_SUMMARY.txt.jinja2

```diff
@@ -0,0 +1,27 @@
+---------------------------------------------------------
+
+This is summary of the cherry-picks:
+   Previous release: {{ previous_release }}
+   Current release: {{ current_release }}
+
+---------------------------------------------------------
+
+Changelog changes {{ changelog_changes | length }}:
+
+{% for change in changelog_changes %}
+   * {{ change.full_hash }} : {{ change.message }}
```

Comment:
```suggestion
   * {{ change.short_hash }} : {{ change.message }} : http://github.com/apache/airflow/pull/{{ change.pr }}
```

When going through the summary, I found clickable links really helpful.

Was `short_hash` not unique enough?

Reply:
Yeah. I also think links are perfect!. 

And yeah, short hash should be unfque enough. 

The problem with short hashes (not everyone knows that) that they are variable length - it's length depends on:

* git version
* how many commits you have in repo
* how long ago you did garbage collection (the non-garbage collected commits also count when git determines whether 
short hash is "unique enough"

So I tend to use long hashes when I am going to use them for anything, simply because they are predictable length.

However in this case, what we are going to to do is to block-copy the hashes and the length of short hash is not going to change in one block (depending on what you run in parallel or whether garbage collection kicks-in, it might change betweeen blocks). But it does not matter in this case.

Fun fact - it ACTUALLY HAPPENED to me when last month I prepared provider documentation, and the hash went back to 1-character shorter when I run garbage collection so I know we are really close to the treshold where it will get longer permanently. This is how I know short hash is variable in length https://github.com/git/git/commit/bb188d00f7996639e65159f1467758e66fcd084b

Adding URL make the line already long so short hash makes more sense.